app/views/candidate_interface/subject_knowledge/show.html.erb

```diff
@@ -5,11 +5,25 @@
   <%= t('page_titles.subject_knowledge') %>
 </h1>
 
+<% if @application_form.review_pending?(:subject_knowledge) %>
+  <div class='govuk-inset-text govuk-!-width-two-thirds govuk-!-margin-top-0'>
+    <h2 class="govuk-heading-m">Feedback from previous applications</h2>
```

Comment:
I think it makes sense to pluralise this based on the number of bits of feedback shown?

```suggestion
    <h2 class="govuk-heading-m">Feedback from previous <%= pluralize(@application_form.subject_knowledge_rejection_reasons.size, "application") %></h2>
```